src/components/FloatingBtn.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import '../css/FloatingBtn.css';
+import { useNavigate } from 'react-router-dom';
+import sunny from '../assets/img/sunny.png';
+
+export default function FloatingBtn({ }) {
+    const navigate = useNavigate();
+    const onClickBtn = () => {
+        navigate('/cloudy/letter');
+    }
+
+    return (
+      <button className='floating-button' onClick={(onClickBtn)}>
+        <img src={sunny} alt=' ' /> 
```

Comment:
```suggestion
        <img src={require('../assets/img/sunny.png)} alt='' /> 
```
require로 통일하거나 import로 통일하거나 하나로 통일 필요할 것 같아요
공백 지워 주세요!